tests/testutils/collector_process.go

```diff
@@ -49,6 +49,10 @@ type CollectorProcess struct {
 	Fail             bool
 }
 
+func (collector CollectorProcess) WithBoundDirectory(path string, mountPoint string) (Collector, error) {
+	return &collector, ErrUnsupportedFeature
```

Comment:
@rmfitzpatrick I expect there is a better way that will be obvious to you - please let me know.